doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -267,206 +267,270 @@ If the cluster has started but an OSD isn't starting, check the following:
 An OSD Failed
 -------------
 
-When a ``ceph-osd`` process dies, surviving ``ceph-osd`` daemons will report
-to the mons that it appears down, which will in turn surface the new status
-via the ``ceph health`` command::
+When an OSD has failed, this means that a ``ceph-osd`` process has died and
```

Comment:
s/has failed/fails/
s/has died/is unresponsive or has died/

Reply:
Accepted.